shared-module/is31fl3741/is31fl3741.h

```diff
@@ -0,0 +1,161 @@
+    184,   182,   183,   // (2,4)
+    37,    35,    36,    // (3,0)
+    40,    38,    39,    // (3,1)
+    43,    41,    42,    // (3,2)
+    46,    44,    45,    // (3,3)
+    34,    32,    33,    // (3,4)
+    67,    65,    66,    // (4,0)
+    70,    68,    69,    // (4,1)
+    73,    71,    72,    // (4,2)
+    76,    74,    75,    // (4,3)
+    64,    62,    63,    // (4,4)
+    97,    95,    96,    // (5,0)
+    100,   98,    99,    // (5,1)
+    103,   101,   102,   // (5,2)
+    106,   104,   105,   // (5,3)
+    94,    92,    93,    // (5,4)
+    127,   125,   126,   // (6,0) / right ring pixel 3
+    130,   128,   129,   // (6,1)
+    133,   131,   132,   // (6,2)
+    136,   134,   135,   // (6,3)
+    124,   122,   123,   // (6,4)
+    157,   155,   156,   // (7,0)
+    160,   158,   159,   // (7,1)
+    163,   161,   162,   // (7,2) / right ring pixel 5
+    166,   164,   165,   // (7,3) / 6
+    244,   242,   243,   // (7,4) / 7
+    247,   245,   246,   // (8,0)
+    250,   248,   249,   // (8,1)
+    253,   251,   252,   // (8,2)
+    256,   254,   255,   // (8,3)
+    65535, 65535, 65535, // (8,4) (clipped, nose bridge)
+    345,   347,   346,   // (9,0)
+    342,   344,   343,   // (9,1)
+    267,   269,   268,   // (9,2)
+    263,   265,   264,   // (9,3)
+    65535, 65535, 65535, // (9,4) (clipped, nose bridge)
+    336,   338,   337,   // (10,0)
+    333,   335,   334,   // (10,1)
+    237,   239,   238,   // (10,2) / left ring pixel 19
+    233,   235,   234,   // (10,3) / 18
+    348,   262,   349,   // (10,4) / 17
+    327,   329,   328,   // (11,0) / left ring pixel 21
+    324,   326,   325,   // (11,1)
+    207,   209,   208,   // (11,2)
+    203,   205,   204,   // (11,3)
+    330,   202,   331,   // (11,4)
+    318,   320,   319,   // (12,0)
+    315,   317,   316,   // (12,1)
+    177,   179,   178,   // (12,2)
+    173,   175,   174,   // (12,3)
+    321,   172,   322,   // (12,4)
+    309,   311,   310,   // (13,0)
+    306,   308,   307,   // (13,1)
+    147,   149,   148,   // (13,2)
+    143,   145,   144,   // (13,3)
+    312,   142,   313,   // (13,4)
+    300,   302,   301,   // (14,0)
+    297,   299,   298,   // (14,1)
+    117,   119,   118,   // (14,2)
+    113,   115,   114,   // (14,3)
+    303,   112,   304,   // (14,4)
+    291,   293,   292,   // (15,0)
+    288,   290,   289,   // (15,1)
+    87,    89,    88,    // (15,2)
+    83,    85,    84,    // (15,3)
+    294,   82,    295,   // (15,4)
+    282,   284,   283,   // (16,0) / left ring pixel 3
+    279,   281,   280,   // (16,1)
+    57,    59,    58,    // (16,2)
+    53,    55,    54,    // (16,3)
+    285,   52,    286,   // (16,4)
+    65535, 65535, 65535, // (17,0) (clipped, corner)
+    270,   272,   271,   // (17,1) / left ring pixel 4
+    27,    29,    28,    // (17,2) / 5
+    23,    25,    24,    // (17,3) / 6
+    276,   22,    277,   // (17,4) / 7
+};
+
+// Gamma correction table
+static const uint8_t IS31GammaTable[256] = {
```

Comment:
Would it make sense to calculate this at run time (using RAM) vs compile time? A bonus of run-time is the gamma level could be user controller.

Reply:
If we want user control then RAM is a good choice. I'd start simpler and just hard code it like you did here. The generation code will also take space so the savings will be < 256 bytes. Simplicity is worth whatever the difference is.